amazon-kinesis-client/src/test/java/software/amazon/kinesis/retrieval/fanout/FanOutRecordsPublisherTest.java

```diff
@@ -1086,7 +1086,7 @@ public void testContinuesAfterSequence() {
                         .type(ShardIteratorType.AT_SEQUENCE_NUMBER).build())
                 .build();
 
-        verify(kinesisClient).subscribeToShard(eq(expected), flowCaptor.capture());
+        verify(kinesisClient).subscribeToShard(any(SubscribeToShardRequest.class), flowCaptor.capture());
```

Comment:
This makes the verification/test pretty useless since we cannot tell where we are trying to start at. If we cannot verify the position here, can we verify elsewhere which sequence number we are attempting to start at?

See final comment for suggestion for new argument matcher to confirm only the fields we care about equal, we don't really care about object equality in any of these tests. (this can be applied to most edits in this commit if we decide we continue to care about these verifications)